tests/stub/routing/test_routing_v4x4.py

```diff
@@ -2448,3 +2448,100 @@ def test_should_fail_when_driver_closed_using_session_run(self):
         self.assertEqual(summary.server_info.address,
                          get_dns_resolved_server_address(self._readServer1))
         self.assertEqual([1], sequence)
+
+    def test_should_fail_when_writing_without_writers_using_session_run(self):
```

Comment:
To me it seems like this test is just a subset of `test_should_serve_reads_and_fail_writes_when_no_writers_available`

Reply:
Yes, it is closely related. However, it uses a new session for writes immediately and more closely resembles the original IT test that I am going to replace with it. It would be nice if we could have it here.

Reply:
:ok: